evaluation/logic_reasoning/instruction.txt

```diff
@@ -0,0 +1,20 @@
+You are a helpful assistant assigned with logic reasoning task. You need to determine the correctness of a query given some facts and fules. 
+you can interact with an interactive Python (Jupyter Notebook) environment and receive the corresponding output when needed. You first need to install a python package through ```pip install scitools-pyke```. The code should be enclosed using "<execute_ipython>" tag.
```

Comment:
One tips: we have a `sandbox` parameter in main function, and you can execute the installation at here, then it may save some cost when you call gpt. At least they can reduce one action. But it is also fine to tell gpt in instructions.

Reply:
thanks for the tip, any example for this? byw, currently i deprecated the sandbox in the main function.

Reply:
Here's an example:

https://github.com/ryanhoangt/OpenDevin/blob/dce15b51237a7e9bbd399158a1c0c6536937f7d3/evaluation/mint/run_infer.py#L118-L130

Reply:
Thanks! already integrated! Sandbox is awesome, saving one action step for each instance!